Code/GraphMol/MolDraw2D/MolDraw2DHelpers.h

```diff
@@ -207,6 +207,8 @@ struct RDKIT_MOLDRAW2D_EXPORT MolDrawOptions {
                                      // mean bond length
   double padding =
       0.05;  // fraction of empty space to leave around the molecule
+  double reagentPadding = 0.0;  // fraction of empty space to leave around
```

Comment:
isn't `reagent` only the reactants and maybe agents?
Maybe this should be `componentPadding`?

Reply:
I guess that makes sense. It does put padding round the agents over the arrow as well.